<!DOCTYPE html>
<html class="writer-html4" lang="en" >
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>submodule &mdash; moder v0 documentation</title><link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  <script id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script type="text/javascript" src="_static/jquery.js"></script>
        <script type="text/javascript" src="_static/underscore.js"></script>
        <script type="text/javascript" src="_static/doctools.js"></script>
        <script type="text/javascript" src="_static/language_data.js"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="example" href="example.html" />
    <link rel="prev" title="Installation" href="installation.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
            <a href="index.html" class="icon icon-home"> moder
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">submodule</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#module1-expression-data-analysis">module1: Expression Data Analysis</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="#module2-splicing-data-analysis">module2: Splicing Data Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="#module3-allele-specific-expression-analysis">module3: Allele Specific Expression Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="example.html">example</a></li>
<li class="toctree-l1"><a class="reference internal" href="example.html#input">INPUT</a></li>
<li class="toctree-l1"><a class="reference internal" href="example.html#eoutliers">eOutliers</a></li>
<li class="toctree-l1"><a class="reference internal" href="example.html#soutliers">sOutliers</a></li>
<li class="toctree-l1"><a class="reference internal" href="example.html#aseoutliers">aseOutliers</a></li>
<li class="toctree-l1"><a class="reference internal" href="example.html#visualize">VISUALIZE</a></li>
<li class="toctree-l1"><a class="reference internal" href="license.html">License</a></li>
<li class="toctree-l1"><a class="reference internal" href="version.html">version</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">moder</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
      <li>submodule</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/submodule.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <div class="section" id="submodule">
<h1>submodule<a class="headerlink" href="#submodule" title="Permalink to this headline">¶</a></h1>
<div class="section" id="module1-expression-data-analysis">
<h2>module1: Expression Data Analysis<a class="headerlink" href="#module1-expression-data-analysis" title="Permalink to this headline">¶</a></h2>
<p>This module is designed to analysis gene expression data. The basic command line arguments and descriptions as follows. More available parameters refer to <a class="reference external" href="http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf">RNA-SeQC</a> and <a class="reference external" href="https://github.com/gonglihai-hub/mOutlierPipe/blob/singleTissue/doc/peer.md">PEER</a></p>
<p>Look up <a href="#id1"><span class="problematic" id="id2">`example for eOutliers &lt;&gt;`_</span></a> for more infomation about how to use this submodule.</p>
<p><strong>command line arguments</strong></p>
<table border="1" class="docutils">
<colgroup>
<col width="10%" />
<col width="90%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">args</th>
<th class="head">description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>–expression</td>
<td>assign mode to analysis Gene Expression data</td>
</tr>
<tr class="row-odd"><td>-i,–input</td>
<td>txt file with all input bam file path (required)</td>
</tr>
<tr class="row-even"><td>–gtf</td>
<td>genome annotation file in GTF format (required)</td>
</tr>
<tr class="row-odd"><td>-o,–output</td>
<td>directory to store all resulting files&lt;br&gt;&lt;font color=’red’&gt;(optional and default output dir is current directory)&lt;/font&gt;</td>
</tr>
<tr class="row-even"><td>-p,–parallel</td>
<td>parallel number &lt;br&gt;&lt;font color=’red’&gt;(optional and defalut value is 1)&lt;/font&gt;</td>
</tr>
<tr class="row-odd"><td>–threshold</td>
<td>threshold of z_score, used to filter results’ value larger than threshold&lt;br&gt;&lt;font color=’red’&gt;(optional and default value is 2)&lt;/font&gt;</td>
</tr>
</tbody>
</table>
<p><strong>running example</strong></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">python</span> <span class="n">mOutlierPipe</span><span class="o">.</span><span class="n">py</span> <span class="o">--</span><span class="n">expression</span>
        <span class="o">--</span><span class="n">parallel</span> <span class="mi">8</span>
        <span class="o">--</span><span class="nb">input</span> <span class="n">file_path</span><span class="o">.</span><span class="n">txt</span>
        <span class="o">--</span><span class="n">gtf</span> <span class="n">sample_annotation</span><span class="o">.</span><span class="n">gtf</span>
        <span class="o">--</span><span class="n">threshold</span> <span class="mi">2</span>
</pre></div>
</div>
</div>
</div>
<div class="section" id="module2-splicing-data-analysis">
<h1>module2: Splicing Data Analysis<a class="headerlink" href="#module2-splicing-data-analysis" title="Permalink to this headline">¶</a></h1>
<p>This module is designed to analysis splicing data. The basic command line arguments and descriptions as follows. More available parameters refer to <a class="reference external" href="https://github.com/gonglihai-hub/mOutlierPipe/blob/singleTissue/doc/leafcutter.md">leafcutter</a>, <cite>SPOT &lt;https://github.com/gonglihai-hub/mOutlierPipe/blob/singleTissue/doc/SPOT.md&gt;</cite></p>
<p>Look up <a href="#id3"><span class="problematic" id="id4">`example for sOutliers &lt;&gt;`_</span></a> for more infomation about how to use this submodule.</p>
<p><strong>command line arguments</strong></p>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">args</th>
<th class="head">description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>–splicing</td>
<td>assign mode to analysis Splicing data</td>
</tr>
<tr class="row-odd"><td>-i , –input</td>
<td>txt file with all input bam file path (required)</td>
</tr>
<tr class="row-even"><td>–gtf</td>
<td>genome annotation file in GTF format, used to translate cluster id to gene id (required)</td>
</tr>
<tr class="row-odd"><td>-o , –output</td>
<td>directory to store all resulting files&lt;br&gt;&lt;font color=’red’&gt;(optional and default output dir is current directory)&lt;/font&gt;</td>
</tr>
<tr class="row-even"><td>-p , –parallel</td>
<td>parallel number &lt;br&gt;&lt;font color=’red’&gt;(optional and default value is 1)&lt;font&gt;</td>
</tr>
<tr class="row-odd"><td>–threshold</td>
<td>threshold of z_score, in splicing analysis pipeline, the value of z will be translated to p&lt;br&gt;&lt;font color=’red’&gt;(optional and default value is 0.0027)&lt;font&gt;</td>
</tr>
</tbody>
</table>
<blockquote>
<div><strong>running example</strong></div></blockquote>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">python</span> <span class="n">mOutlierPipe</span><span class="o">.</span><span class="n">py</span> <span class="o">--</span><span class="n">splicing</span>
        <span class="o">--</span><span class="n">parallel</span> <span class="mi">8</span>
        <span class="o">--</span><span class="nb">input</span> <span class="n">file_path</span><span class="o">.</span><span class="n">txt</span>
        <span class="o">--</span><span class="n">gtf</span> <span class="n">genome_annotation</span><span class="o">.</span><span class="n">gtf</span>
        <span class="o">--</span><span class="n">threshold</span> <span class="mi">2</span>
</pre></div>
</div>
</div>
<div class="section" id="module3-allele-specific-expression-analysis">
<h1>module3: Allele Specific Expression Analysis<a class="headerlink" href="#module3-allele-specific-expression-analysis" title="Permalink to this headline">¶</a></h1>
<p>This module is designed to analysis allele specific expression data. The basic command line arguments and descriptions as follows. More available parameters refer to <a class="reference external" href="https://github.com/Xu-Dong/MODER/blob/singleTissue/docs/args/phASER.md">phASER</a></p>
<p>Look up <a href="#id5"><span class="problematic" id="id6">`example for aseOutliers &lt;&gt;`_</span></a> for more infomation about how to use this submodule.</p>
<p><strong>command line arguments</strong></p>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">args</th>
<th class="head">description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>–ase</td>
<td>assign mode to analysis ASE data</td>
</tr>
<tr class="row-odd"><td>-i , –input</td>
<td>txt file with all input bam file path (required)</td>
</tr>
<tr class="row-even"><td>–gtf</td>
<td>genome annotation file in GTF format, used to translate cluster id to gene id (required)</td>
</tr>
<tr class="row-odd"><td>–vcf</td>
<td>Variant Call Format file, include genetic variation in regulation in a population about the genome (required)</td>
</tr>
<tr class="row-even"><td>–variant</td>
<td>tissue-specific estimates of genetic variation in gene dosage (required)</td>
</tr>
<tr class="row-odd"><td>-o , –output</td>
<td>directory to store all resulting files&lt;br&gt;&lt;font color=’red’&gt;(optional and default output dir is current directory)&lt;/font&gt;</td>
</tr>
<tr class="row-even"><td>-p , –parallel</td>
<td>parallel number &lt;br&gt;&lt;font color=’red’&gt;(optional and default value is 1)&lt;font&gt;</td>
</tr>
<tr class="row-odd"><td>–threshold</td>
<td>threshold of z_score, in ase analysis pipeline, the value of z will be translated to p&lt;br&gt;&lt;font color=’red’&gt;(optional and default value is 0.0027)&lt;font&gt;</td>
</tr>
<tr class="row-even"><td>–tissue</td>
<td>tissue name in variant file of the sample</td>
</tr>
</tbody>
</table>
<p><strong>running example</strong></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">python</span> <span class="n">mOutlierPipe</span><span class="o">.</span><span class="n">py</span> <span class="o">--</span><span class="n">ase</span>
        <span class="o">--</span><span class="n">parallel</span> <span class="mi">8</span>
        <span class="o">--</span><span class="nb">input</span> <span class="n">file_path</span><span class="o">.</span><span class="n">txt</span>
        <span class="o">--</span><span class="n">gtf</span> <span class="n">genome_annotation</span><span class="o">.</span><span class="n">gtf</span>
        <span class="o">--</span><span class="n">vcf</span> <span class="n">sample</span><span class="o">.</span><span class="n">vcf</span>
        <span class="o">--</span><span class="n">variant</span> <span class="n">Vg_GTEx_v8</span><span class="o">.</span><span class="n">txt</span>
        <span class="o">--</span><span class="n">threshold</span> <span class="mi">2</span>
        <span class="o">--</span><span class="n">tissue</span> <span class="n">MSCLSK</span>
</pre></div>
</div>
</div>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="installation.html" class="btn btn-neutral float-left" title="Installation" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="example.html" class="btn btn-neutral float-right" title="example" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2021, GongLihai.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>